src/main/kotlin/no/nav/tilleggsstonader/sak/statistikk/behandling/BehandlingsstatistikkService.kt

```diff
@@ -0,0 +1,204 @@
+
+        return BehandlingDVH(
+            behandlingId = saksbehandling.eksternId.toString(),
+            behandlingUuid = behandlingId.toString(),
+            sakId = saksbehandling.eksternId.toString(),
+            aktorId = saksbehandling.ident,
+            registrertTid = saksbehandling.opprettetTid.atZone(ZONE_ID_OSLO)
+                ?: henvendelseTidspunkt.atZone(ZONE_ID_OSLO),
+            endretTid = hendelseTidspunkt.atZone(ZONE_ID_OSLO),
+            tekniskTid = zonedNow(),
+            behandlingStatus = hendelse.name,
+            opprettetAv = maskerVerdiHvisStrengtFortrolig(
+                strengtFortroligAdresse,
+                saksbehandlerId,
+            ),
+            saksnummer = saksbehandling.eksternFagsakId.toString(),
+            mottattTid = henvendelseTidspunkt.atZone(ZONE_ID_OSLO),
+            saksbehandler = maskerVerdiHvisStrengtFortrolig(
+                strengtFortroligAdresse,
+                saksbehandlerId,
+            ),
+            ansvarligEnhet = maskerVerdiHvisStrengtFortrolig(
+                strengtFortroligAdresse,
+                sisteOppgaveForBehandling?.tildeltEnhetsnr ?: MASKINELL_JOURNALFOERENDE_ENHET,
+            ),
+            behandlingMetode = behandlingMetode?.name ?: "MANUELL",
+            behandlingÅrsak = saksbehandling.årsak.name,
+            avsender = "NAV Tilleggstønader",
+            behandlingType = BehandlingType.valueOf(saksbehandling.type.name).toString(),
+            sakYtelse = saksbehandling.stønadstype.name,
+            behandlingResultat = saksbehandling.resultat.name,
+            resultatBegrunnelse = null, // TODO er fritekstfelt, og er ikke ønsket i statistikk før enum er implementert
+            ansvarligBeslutter =
+            if (!beslutterId.isNullOrEmpty()) {
+                maskerVerdiHvisStrengtFortrolig(
+                    strengtFortroligAdresse,
+                    beslutterId.toString(),
+                )
+            } else {
+                null
+            },
+            vedtakTid = if (Hendelse.VEDTATT == hendelse) {
+                hendelseTidspunkt.atZone(ZONE_ID_OSLO)
+            } else {
+                null
+            },
+            ferdigBehandletTid = if (Hendelse.FERDIG == hendelse) {
+                hendelseTidspunkt.atZone(ZONE_ID_OSLO)
+            } else {
+                null
+            },
+            totrinnsbehandling = totrinnskontrollErGodkjent(behandlingId),
+            sakUtland = mapTilStreng(saksbehandling.kategori),
+            relatertBehandlingId = relatertEksternBehandlingId,
+            versjon = Applikasjonsversjon.versjon,
+            vilkårsprøving = emptyList(), // TODO: Implementer dette i samarbeid med Team SAK. Ikke kritisk å ha med i starten.
+            revurderingÅrsak = null, // TODO aktiver når revurdering er implementert
+            revurderingOpplysningskilde = null, // TODO aktiver når revurdering er implementert
+        )
```

Comment:
`totrinnskontrollService.hentX(behandlingId)` hentes 3 ganger, 
Er det tilstrekkelig å hente den 1 gang? Bruk gjerne `hentTotrinnskontroll`

Noen av kallene gjør en del logikk som er unødvendig i TotrinnskontrollService som vi ikke trenger

Reply:
Fikk kortet ned til to kall. `hentSaksbehandlerSomSendteTilBeslutter` lå ikke (såvidt vi kan se) på `totrinnskontroll`-objektet